### ISSUE TITLE
Try to get the vocab of NestedField but only get '[]'

### ISSUE THREAD

**Issue by Wuyou-HIC** [pytorch/text#371]
**- Here is the dataset**
----------------------
train1.csv: "3","EBay gets into rentals","EBay plans to buy the apartment and home rental service Rent.com for \$415 million, adding to its already exhaustive breadth of offerings."
test1.csv: "3","EBay gets into rentals","EBay plans to buy the apartment and home rental service Rent.com for \$415 million, adding to its already exhaustive breadth of offerings."
----------------------
**- Here is my code**
----------------------
import os
import torch
import torchtext
from torchtext import data

import spacy 
spacy_en = spacy.load('en')

#tokenize_simple = lambda x: x.split()

def tokenizer(text):# create a tokenizer function
    # 返回 a list of <class 'spacy.tokens.token.Token'>
    return [tok.text for tok in spacy_en.tokenizer(text)]
    
def sentenizer(text):
	return [sent.text for sent in spacy_en(text).sents]

# Field for label
LABEL = data.Field(sequential=False, use_vocab=False)

# Field for text_1d
TEXT_1d = data.Field(sequential=True, tokenize=tokenizer, lower=True) #fix_length = 90
#TEXT_rnn = data.ReversibleField(sequential=True, lower=True, include_lengths=True) #tokenize=tokenizer_1d | sort_within_batch=True

# Field for text_2d
SENTENCE = data.Field(sequential=True, tokenize=sentenizer, lower=True)
TEXT_NESTING = data.Field(sequential=True, tokenize=tokenizer, lower=True)
TEXT_2d = data.NestedField(TEXT_NESTING)

# Field's Combination for preprocessing (1d)
zhang2015_datafields_1d = [('Label', LABEL), ('Text', TEXT_1d)]

# Field's Combination for preprocessing (2d)
zhang2015_datafields_2d = [('Label', LABEL), (('Sentence', 'Text'), (SENTENCE, TEXT_2d))]

#########################################################################################

train, test = data.TabularDataset.splits(path='data/ag_news_csv/', \
	train='train1.csv', \
	test='test1.csv', \
	format='csv', \
	fields=zhang2015_datafields_2d)

print(train.fields)
print(len(train))
print(vars(train[0]))

TEXT_2d.build_vocab(train)

print (TEXT_2d.vocab.freqs.most_common(10))
----------------------
**- Here is my output**
----------------------
{'Text': <torchtext.data.field.NestedField object at 0x118871208>, 'Sentence': <torchtext.data.field.Field object at 0x1188714e0>, 'Label': <torchtext.data.field.Field object at 0x118871518>}
1
{'Sentence': ['ebay gets into rentals'], 'Text': [['ebay'], ['gets'], ['into'], ['rentals']], 'Label': '3'}
[]
